docs/release-notes/cli-cmd-describe.rst

```diff
@@ -0,0 +1,5 @@
+:orphan:
```

Comment:
add a bug fix note about Fix a bug n error related to det cmd ls --csv

e.g., 

**Bug Fixes**

- CLI: Fix a bug where....